Automattic/jetpack
# My Jetpack: Update/my jetpack restore get product resolver

This PR restores the `getProduct()` resolver in order to anticipate issues for those products that will change their state asynchronically. I've removed the impact in the UI when requesting data, though.
We can keep improving the process in follow-up PRs, for instance, defining which product needs to hit the backend to get a fresh status.

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: Update/my jetpack restore get product resolver

#### Jetpack product discussion


p1643743260988689?thread_ts=1643733211.545209&cid=C02TQF5VAJD-slack-

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






You shouldn't see any visual change in the UI.
* Go to My Jetpack
* Confirm the Product cards are properly rendered
* Activate/deactivate products
* You can take a look at the network tab to see that the client still performs requests
